### PR TITLE
led brightness argument

### PR DESCRIPTION
**Pull request by llluis**
Second try
Request #98.

### CODE REVIEW

**Comment by blacknell** [comment]
this is great - much needed

**Comment by Pr0mises** [comment]
works great thank you

**Comment by Aohzan** [comment]
hello @synesthesiam , can you check this PR :pray:

**Comment by TN-1** [comment]
Working brilliantly, this is much needed in master. Can it be merged in please? Is there any work that is required?

File: examples/2mic_service.py [hunk 1]
```diff
@@ -41,8 +41,8 @@ async def main() -> None:
     """Main entry point."""
     parser = argparse.ArgumentParser()
     parser.add_argument("--uri", required=True, help="unix:// or tcp://")
-    #
     parser.add_argument("--debug", action="store_true", help="Log DEBUG messages")
+    parser.add_argument("--led-brightness", type=int, default=31, help="LED brightness (integer from 1 to 31)")
```
Comment (synesthesiam) [inline examples/2mic_service.py:45]: Can you add `choices=range(1, 32)` to restrict the range?

**Review by synesthesiam — CHANGES_REQUESTED** [review]

**Comment by synesthesiam** [comment]
Small change requested and we're good to go 👍

**Review by synesthesiam — APPROVED** [review]

**Comment by synesthesiam** [comment]
Merging now. Will add changes afterwards.

**Comment by huseman21** [comment]
I'm doing somthing wrong,  This is how I am calling Wyoming-satellite but it errors out with the --led-brightness. any help?

[Service]
Type=simple
ExecStart=/home/me/wyoming-satellite/script/run --name 'my satellite' --led-brightness 5 --uri 'tcp://0.0.0.0:10700' --mic-command 'arecord -D plughw:CA>
WorkingDirectory=/home/huseman/wyoming-satellite
Restart=always
RestartSec=8

**Comment by Aohzan** [comment]
It's an other service to run, my config (from 1 year ago)

```
[Unit]
Description=Respeaker LEDs

[Service]
Type=simple
ExecStart=/home/me/wyoming-satellite/examples/.venv/bin/python3 2mic_service.py --uri 'tcp://127.0.0.1:10500' --led-brightness 3
WorkingDirectory=/home/me/wyoming-satellite/examples
Restart=always
RestartSec=10

[Install]
WantedBy=default.target
```

**Pull request merged**